Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -414,13 +414,32 @@ extension SourceFilesList {
 
 extension ResourceFileElements {
     fileprivate static func from(resources: [PackageInfo.Target.Resource], path: AbsolutePath) -> Self? {
-        let resourcesPaths = resources.map { path.appending(RelativePath($0.path)) }
+        var resourcesPaths = resources.map { path.appending(RelativePath($0.path)) }
+        resourcesPaths.append(contentsOf: defaultResources(from: path))
```

Comment:
Are you sure those are added even if you add a custom `resources` parameter in the Package.swift?

Reply:
From what I see [here](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#resource) looks like the logic is correct